openedx/features/content_type_gating/helpers.py

```diff
@@ -17,6 +17,14 @@
     OrgInstructorRole,
     GlobalStaff
 )
+from xmodule.partitions.partitions import Group
+
+CONTENT_TYPE_GATE_GROUP_IDS = {
+    'limited_access': 1,
+    'full_access': 2,
+}
+LIMITED_ACCESS = Group(CONTENT_TYPE_GATE_GROUP_IDS['limited_access'], 'Limited-access Users')
+FULL_ACCESS = Group(CONTENT_TYPE_GATE_GROUP_IDS['full_access'], 'Full-access Users')
```

Comment:
Does moving these out to the `helpers.py` file help prevent a circular import situation?